scram-tools.file/tools/cepgen/cepgen.xml

```diff
@@ -0,0 +1,19 @@
+<tool name="cepgen" version="@TOOL_VERSION@">
+  <info url="https://cepgen.hepforge.org/"/>
+  <use name="hepmc"/>
+  <use name="hepmc3"/>
+  <use name="lhapdf"/>
+  <use name="pythia6"/>
+  <lib name="CepGen"/>
+  <lib name="CepGenHepMC2"/>
+  <lib name="CepGenHepMC3"/>
+  <lib name="CepGenLHAPDF"/>
+  <lib name="CepGenPythia6"/>
+  <client>
+    <environment name="CEPGEN_DIR" default="@TOOL_ROOT@"/>
+    <environment name="LIBDIR" default="$CEPGEN_DIR/lib"/>
+    <environment name="INCLUDE" default="$CEPGEN_DIR/include"/>
+  </client>
+  <runtime name="PATH" value="$CEPGEN_DIR/bin" type="path"/>
+  <runtime name="CEPGEN_PATH" value="$CEPGEN_DIR/share/CepGen"/>
```

Comment:
please replace all `CEPGEN_DIR` with `CEPGEN_BASE` in tool file

Reply:
Sorry, I realise I did not change it in the first place. Done with 79e667e29.